addons/io/org.openhab.io.openhabcloud/src/main/java/org/openhab/io/openhabcloud/internal/CloudClient.java

```diff
@@ -588,36 +588,36 @@ public void run() {
                         socket.emit("responseFinished", responseJson);
                         logger.debug("Finished responding to request {}", mRequestId);
                     } catch (JSONException e) {
-                        logger.error(e.getMessage());
+                        logger.error("{}", e.getMessage());
                     }
                 }
             }, 1, TimeUnit.MILLISECONDS);
         }
 
         @Override
         public synchronized void onFailure(Request request, Throwable failure) {
-            logger.error(failure.getMessage());
+            logger.error("{}", failure.getMessage());
             JSONObject responseJson = new JSONObject();
             try {
                 responseJson.put("id", mRequestId);
                 responseJson.put("responseStatusText", "openHAB connection error: " + failure.getMessage());
                 socket.emit("responseError", responseJson);
             } catch (JSONException e) {
-                logger.error(e.getMessage());
+                logger.error("{}", e.getMessage());
```

Comment:
I would log the exception instead, it is an error, I want to be able to debug the stacktrace